_posts/2018-11-27-linguistic diversity and Ad-hoc translation of the Programming Historian’s lessons.md

```diff
@@ -0,0 +1,41 @@
+
+On the other hand, if you are you interested in translating a (part of a) Programming Historian lesson, one that you ’ve found e.g. particularly useful for your community of practice or for the class you are teaching, the procedure is much more easier and we actively support and encourage such ad-hoc translation initiatives. Needless to say, all of our lessons are published under the Creative Commons ‘Attribution’ licence (CC BY) and this allows others to distribute, remix, reuse and build upon the licensed work as long as the original source is credited. By choosing the most liberal of the CC licences (and taking the risk of use within the for-profit sector) we allow derivative works of the lessons, including translations, and we enable onward reuse.
+
+**How to embark on an ad-hoc translation?**
+- Choose a lesson you want to translate in a different language. As a starting point, you can either use the online version of the lesson or the md file from the github repo of the project.
+-	You can always consult our [Author](https://programminghistorian.org/en/author-guidelines) or [Editor](https://programminghistorian.org/en/editor-guidelines) guidelines to get the spirit of our lessons’ structure, wording and overall style.
+-	As ad-hoc translations are a spin-off of the PH and they are not (necessarily) need to fit to our infrastructure, the easiest and more obvious way to do it, is to translate it in using your own infrastructure and services, in order to fit your original purposes.
+1.	Once you have a final version of the translation and you want to share it with the project, please do send us the file (preferably if you have a md file). We then going to customize the lesson and upload it as a translation version of the original one.
+2.	If you don’t want to share the translation with us, please do provide us with a bit of information about the translation work itself (translators, institution, publication, purpose) for our own records and do let us know if you want us to place a small note about the translation at the lesson’s page.
+
+**Why to embark on an ad-hoc translation ?**
+Here are a couple of ideas 
+-	As part of your class assignment 
+o	students familiarise themselves with the methods- tools in question
+o	translation challenges towards terminology 
+-	As part of an edit-a-thon organised in your country/institution – invite us or let us know!
+-	As an one – off contribution of a particularly interesting lesson for your local community
+
+**and a couple of examples**
```

Comment:
are there supposed to be examples here?

Reply:
yes, but I m still waiting for team's input here !